packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -352,9 +373,11 @@ export default class ImageEdit implements EditorPlugin {
         this.lastSrc = this.image.getAttribute('src');
 
         // Set image src to original src to help show editing UI, also it will be used when regenerate image dataURL after editing
+        this.image.style.visibility = 'hidden';
```

Comment:
So this will still modify the editor content.

Can we add this visibility rule to image selection's global CSS so that editor content is not impacted?